screens/Group/GroupDetailScreen.js

```diff
@@ -292,29 +293,181 @@ const styles = StyleSheet.create({
     marginTop: 40,
   },
 });
+const diff = (obj1, obj2) => {
```

Comment:
These utility functions are duplicated in the two detail screens. Could they be moved to a shared file to prevent duplicate code and to clean up the size of these two detail screen files?

Reply:
I created a folder called "shared" with an "index.js" file and added this methods there.